ProfileAPI/ProfileAPI/Services/ProfileService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MongoDB.Driver;
+using ProfileAPI.Configuration;
+using ProfileAPI.Models;
+
+namespace ProfileAPI.Services
+{
+    public class ProfileService
```

Comment:
It would be nice to establish unit tests if it's not too much work.

Reply:
if I have time I might add them tomorrow